### PR TITLE
[autoscaling] Fix and update descheduler

### LINKED ISSUE #7581: [descheduler] cr does not currently reflect all parameters
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Version

v1.57.5

### Expected Behavior

describe all parameters that are available in 
https://github.com/deckhouse/deckhouse/blob/main/modules/400-descheduler/templates/configmap.yaml

### Actual Behavior

cr does not currently reflect all parameters
https://deckhouse.io/documentation/v1/modules/400-descheduler/cr.html
Noticed that there is no such parameters:
`podLifeTime`
`nodeFit`
`namespaceFilter`
`nodeFilter`
`priorityFilter`

### Steps To Reproduce

_No response_

### Additional Information

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
## Description

Fix descheduler CR and update descheduler image. 
Now we supply only two strategies:
* lowNodeUtilization
* highNodeUtilization
Closes #7581

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: autoscaling
type: chore
summary: Fix and update descheduler
impact_level: default
```

### CODE REVIEW

File: modules/400-descheduler/crds/deschedulers.yaml [hunk 1]
```diff
@@ -31,261 +28,298 @@ spec:
                 name:
                   type: string
             spec:
+              type: object
               description: Defines the behavior of a descheduler instance.
               properties:
-                deploymentTemplate:
-                  description: Defines Template of a descheduler Deployment.
+                fitNodesLabelSelector:
```
Comment (nabokihms) [inline modules/400-descheduler/crds/deschedulers.yaml:34]:
```suggestion
                nodeLabelSelector:
```

File: modules/400-descheduler/crds/deschedulers.yaml [hunk 2]
```diff
@@ -31,261 +28,298 @@ spec:
...
+                          - key: tier
+                            operator: NotIn
+                            values: [ production ]
+                    matchLabels:
+                      type: object
                       additionalProperties:
                         type: string
-                      type: object
-                    tolerations:
+                podNamespaceLabelSelector:
```
Comment (nabokihms) [inline modules/400-descheduler/crds/deschedulers.yaml:140]:
```suggestion
                namespaceLabelSelector:
```

File: modules/400-descheduler/crds/deschedulers.yaml (hunk 1 above)
Comment (RomanenkoDenys) [reply modules/400-descheduler/crds/deschedulers.yaml:34]: done

File: modules/400-descheduler/crds/deschedulers.yaml (hunk 2 above)
Comment (RomanenkoDenys) [reply modules/400-descheduler/crds/deschedulers.yaml]: done

**Review by name212 — APPROVED** [review]

**Pull request merged**